### ISSUE TITLE
This is more a wish than a issue.

### ISSUE THREAD

**Issue by KitCatLeo** [jpec/pimp#4]
Thank you for your very practical pimp.py. I am a frequent user of this wonderful script.
To be even happier with it I would like it to have the possibility to create or read from more than one database.
If the command line supported 2 arguments one for the path and the other for the database we would have the possibility to create thematic  playlists, that would be awsome. 
I am not a python programmer I am more in the hardware side. For this reason  a do not dare to do modifications to a script that works very well.
Thank's a lot.
Best regards

**Comment by jpec** [comment]
Hi,

Thank you for your feedback.

Your proposition is interesting and i'll implement it in the next weeks
because I 'm quite busy right now.

Regards,
Julien

**Comment by jpec** [comment]
Hi,

I've added the `--db=/my/path/to/database.file` parameter un pimp.

Please test it !

Julien

**Comment by KitCatLeo** [comment]
I Julien
I am sending my version of pimp.py. After some trial and error  i got it to
work without major flaws.
Since I didn't know nothing about python, I had the opportunity to learn a
bit.
The major problem was with the reload of the database when no path was
given. After reload nothing was displayed and the database was cleaned.
The omxplayer args passing do not seem to work so I decided for a more
direct hardcode approach.
For simplicity I decided to for 2 databases the user database is copied
onto .pimp and .pimp is copied to the user database on exit.
Because of the possibility of definition of a  a path for the database
after the db arg, I was forced to  fix the database name length.
When a user database is indicated and no path is defined the path is
extracted from the first line of the user database.

Best regards
João Mendes

2015-11-09 10:42 GMT+00:00 Julien Pecqueur notifications@github.com:
